src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {

```

Comment:
Don't use verbs for class/interface names. In this case, consider renaming the interface to something like `AreaCalculable` or `AreaProvider` to follow Java naming conventions and to clearly represent the interface's purpose.